### PR TITLE
add to manifest

### PR DESCRIPTION
**Pull request by t3chguy**
for #334
even though app complains about being run in mobile web it is possible to use
and in certain scenarios it might be the best option so this will at least improve that

tested both using Lighthouse and my Android M device

Signed-off-by: Michael Telatynski <7t3chguy@gmail.com>

### CODE REVIEW

File: res/vector-icons/manifest.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
-	"name": "Riot",
+	"name": "Riot - Matrix Client",
```
Comment (lukebarnard1) [inline res/vector-icons/manifest.json]: I'm not convinced by this. @dbkr, how does this sit with you?

File: res/vector-icons/manifest.json [hunk 2]
```diff
@@ -37,5 +41,13 @@
...
 			"density": "4.0"
 		}
-	]
+	],
+    "related_applications": [{
+        "platform": "play",
+        "url": "https://play.google.com/store/apps/details?id=im.vector.alpha",
+        "id": "im.vector.alpha"
+    }, {
+        "platform": "itunes",
```
Comment (lukebarnard1) [inline res/vector-icons/manifest.json]: I wish there was a way to indicate that this isn't a known platform by the w3c. (The w3c claims to only know about "play") but I understand that without the `platform` property, the application isn't processed... Oh well :grinning:

**Comment by lukebarnard1** [comment]
I seem to be getting `Site cannot be installed: no matching service worker detected. You may need to reload the page, or check that the service worker for the current page also controls the start URL from the manifest` when I try adding my local Riot to my Chrome home screen.

**Comment by t3chguy** [comment]
thats strange, definitely worked fine on my Android device

**Comment by lukebarnard1** [comment]
Apparently the manifest has to be in the root directory: https://abendigo.github.io/2016/07/17/pwa_step_one.html#comment-3194480539

**Comment by t3chguy** [comment]
Could you check if you have the same issue at: https://riot.ovh/dev

Lighthouse report: https://riot.ovh/dev/lighthouse.html
only complaint it has is lack of a service worker for app-like behaviour
guess the indexedb worker doesn't quite count

**Comment by t3chguy** [comment]
I guess we could move some of the syncing to a service worker and even get notifications when the webapp isn't open

**Comment by lukebarnard1** [comment]
I think we'll have to do more work to implement PWA things (actually implementing a service worker - https://codelabs.developers.google.com/codelabs/add-to-home-screen/#5). 

I don't want this to block on what to change the name to, so let's stick with just "Riot" and then I'll merge.

**Comment by t3chguy** [comment]
@lukebarnard1 I changed the full name to match that of the apps' `Riot - open team collaboration`

**Comment by lukebarnard1** [comment]
Well in that case LGTM!

**Comment by t3chguy** [comment]
@lukebarnard1 I hadn't updated the PR to match that of my working directory haha

**Comment by lukebarnard1** [comment]
I did not think to check indentation. If you can be bothered to make another PR, go for it :innocent:

**Pull request merged**